packages/node/src/integrations/tracing/nest/sentry-nest-instrumentation.ts

```diff
@@ -100,24 +100,24 @@ export class SentryNestInstrumentation extends InstrumentationBase {
 
             target.prototype.use = new Proxy(target.prototype.use, {
               apply: (originalUse, thisArgUse, argsUse) => {
+                // Middlewares have a request, response and next argument.
+                if (argsUse.length < 3) {
+                  return originalUse.apply(thisArgUse, argsUse);
+                }
```

Comment:
This check is technically redundant with the assertions we have below.

Reply:
Is it? What if the function has 0 arguments?

Reply:
Then an empty array will be passed to `argsUse` and all the arguments we check down below will be falsy (or rather `undefined`).